xdAuthIframe.html

```diff
@@ -0,0 +1,63 @@
+<html>
+    <head></head>
+    <body>
+        <script>
+            var MESSAGE_NAMESPACE = 'kw-xd-auth',
+                authorizedOrigins = [
+                    /^http(s)?:\/\/(.*)(\.)?kano.me$/,
```

Comment:
This origin refer to the frontend apps calling the iframe. Something calling the API won't be affected